cosmovisor/args.go

```diff
@@ -108,21 +113,23 @@ func GetConfigFromEnv() (*Config, error) {
 // and that Name is set
 func (cfg *Config) validate() error {
 	if cfg.Name == "" {
-		return errors.New("DAEMON_NAME is not set")
+		return fmt.Errorf("DAEMON_NAME is not set")
```

Comment:
`errors.New` would be correct here. There isn't anything to format

Reply:
That's the output currently produced by the binary:

```
alessio@phoenix:~/work/cosmos-sdk/cosmovisor$ cmd
DAEMON_NAME is not set
github.com/cosmos/cosmos-sdk/cosmovisor.(*Config).validate
	/home/alessio/work/cosmos-sdk/cosmovisor/args.go:111
github.com/cosmos/cosmos-sdk/cosmovisor.GetConfigFromEnv
	/home/alessio/work/cosmos-sdk/cosmovisor/args.go:100
main.Run
	/home/alessio/work/cosmos-sdk/cosmovisor/cmd/main.go:20
main.main
	/home/alessio/work/cosmos-sdk/cosmovisor/cmd/main.go:11
runtime.main
	/usr/local/go/src/runtime/proc.go:204
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1374
```

And here is the output produced by the new binary:

```
alessio@phoenix:~/work/cosmos-sdk/cosmovisor$ git checkout alessio/cosmovisor-fixes 
Switched to branch 'alessio/cosmovisor-fixes'
Your branch is up-to-date with 'origin/alessio/cosmovisor-fixes'.
alessio@phoenix:~/work/cosmos-sdk/cosmovisor$ make cosmovisor 
go build -mod=readonly ./cmd/cosmovisor
alessio@phoenix:~/work/cosmos-sdk/cosmovisor$ cosmovisor 
DAEMON_NAME is not set
```

IMvHO the latter is best (and enough informative).

Reply:
@marbar3778 please note that the `errors` package used there is `github/pkg/errors`.

Reply:
Could you use "errors" from std?